core/tests/protractor_utils/SkillEditorPage.js

```diff
@@ -120,11 +120,48 @@ var SkillEditorPage = function() {
     });
   };
 
-  this.expectRubricExplanationToMatch = function(index, explanation) {
-    rubricExplanations.then(function(explanations) {
-      explanations[index].getText().then(function(text) {
-        expect(text).toMatch(explanation);
-      });
+  this.deleteRubricExplanationWithIndex = function(diffIndex, explIndex) {
+    var editRubricExplanationButtons = element.all(
+      by.css('.protractor-test-edit-rubric-explanation-' + diffIndex));
+    editRubricExplanationButtons.then(function(buttons) {
+      buttons[explIndex].click();
+      var editor = element(
+        by.css('.protractor-test-rubric-explanation-text'));
+      waitFor.visibilityOf(
+        editor, 'Rubric explanation editor takes too long to appear');
+      deleteRubricExplanationButton.click();
+    });
+  };
+
+  this.editRubricExplanationWithIndex = function(
+    diffIndex, explIndex, explanation) {
+    var editRubricExplanationButtons = element.all(
+      by.css('.protractor-test-edit-rubric-explanation-' + diffIndex));
+    editRubricExplanationButtons.then(function(buttons) {
```

Comment:
ditto: Check the buttons have loaded before clicking

Reply:
Done